Rakefile

```diff
@@ -39,6 +39,7 @@ task :update_graphiql do
 
   FileUtils.mkdir_p(update_path)
   FileUtils.cd(update_path) do
+    sh("npm init --force")
```

Comment:
the `graphiql_update` folder needs a `package.json` file to install the npm packages